src/index.html

```diff
@@ -11,8 +11,62 @@
       rel="stylesheet"
       href="./styles/index.scss"
```

Comment:
Stylesheets with a .scss extension won't work directly in HTML. You should compile them into .css files and reference the compiled .css file here.